api/src/main/java/org/openmrs/module/shr/cdahandler/api/impl/CdaImportServiceImpl.java

```diff
@@ -63,7 +78,7 @@
 	 * @see org.openmrs.module.shr.cdahandler.api.CdaImportService#importDocument(java.io.InputStream)
 	 */
 	@Override
-	public Visit importDocument(InputStream doc) throws DocumentImportException 
+	public Visit importDocument(InputStream doc) throws DocumentImportException
```

Comment:
Bracket should be on the same line as method declaration. Please remove new line here